cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -365,18 +367,33 @@ private static void addRegulatingControlView(Generator g, Map<String, List<Regul
             // PowSyBl has considered the control as continuous and with targetDeadband of size 0
             // The target value is stored in kV by PowSyBl, so unit multiplier is "k"
             String rcid = context.getNamingStrategy().getCgmesIdFromProperty(g, REGULATING_CONTROL_PROPERTY);
+
             double targetDeadband = 0;
+            double target;
+            String targetValueUnitMultiplier;
+            boolean enabled;
+            RemoteReactivePowerControl rrpc = g.getExtension(RemoteReactivePowerControl.class);
+            if (rrpc != null) {
+                target = rrpc.getTargetQ();
+                targetValueUnitMultiplier = "M";
+                enabled = rrpc.isEnabled();
+            } else {
+                target = g.getTargetV();
+                targetValueUnitMultiplier = "k";
+                enabled = g.isVoltageRegulatorOn();
+            }
+
```

Comment:
Use the `getGeneratorRegulatingControlMode` method to obtain the mode, then depending on the mode set the attributes